eoscostarica/guide.eoscostarica.io
# docs(frontend):AgileMethodologies-206

### GH Issue

Resolves #206  

### Steps to test
1. Go to Agile Methodologies.
2. Check content is right.

#### CheckList
- [ ] Add User Stories Description
- [ ] Add Planning Session Description
- [ ] Add Grooming session Description
- [ ] Add Retrospective Session Description
- [ ] Add Demo session description
